src/admin-views/settings/sidebars/default-sidebar.php

```diff
@@ -45,81 +51,104 @@
+			new Link( 'https://evnt.is/1aoz', __( 'Event Tickets Manual', 'event-tickets' ), null, $external_attributes )
+		),
+		( new List_Item( null, null ) )->add_child(
+			new Link( 'https://evnt.is/1axs', __( 'What is Tickets Commerce?', 'event-tickets' ), null, $external_attributes )
+		),
+		( new List_Item( null, null ) )->add_child(
+			new Link( 'https://evnt.is/1axt', __( 'Configuring Tickets Commerce', 'event-tickets' ), null, $external_attributes )
+		),
+		( new List_Item( null, null ) )->add_child(
+			new Link( 'https://evnt.is/1aox', __( 'Using RSVPs', 'event-tickets' ), null, $external_attributes )
+		),
+		( new List_Item( null, null ) )->add_child(
+			new Link( 'https://evnt.is/1aoy', __( 'Managing Orders and Attendees', 'event-tickets' ), null, $external_attributes )
+		),
+	]
 );
 
-$hero_section->add_section(
-	( new Settings_Section() )
-		->add_elements(
-			[
-				( new Paragraph() )->add_child(
-					new Plain_Text( __( 'Having trouble?', 'event-tickets' ) )
-				),
+// Build Advanced Plus Features List.
+$advanced_list = new Unordered_List(
+	new Classes( 'tec-tickets__admin-banner-kb-list' )
+);
 
-				new Link(
-					admin_url( 'edit.php?post_type=tec-tickets-settings&page=tec-tickets-help-hub' ),
-					__( 'Help', 'event-tickets' )
-				),
-				$break,
-				new Link(
-					admin_url( 'admin.php?page=tec-tickets-troubleshooting' ),
-					__( 'Troubleshoot', 'event-tickets' )
-				),
-			]
-		)
+$advanced_list->add_children(
+	[
+		( new List_Item( null, null ) )->add_child(
+			new Link( 'https://evnt.is/1ao-', __( 'Switching from Tribe Commerce to WooCommerce', 'event-tickets' ), null, $external_attributes )
+		),
+		( new List_Item( null, null ) )->add_child(
+			new Link( 'https://evnt.is/1ap0', __( 'Setting Up E-Commerce Plugins for Selling Tickets', 'event-tickets' ), null, $external_attributes )
+		),
+		( new List_Item( null, null ) )->add_child(
+			new Link( 'https://evnt.is/1ap1', __( 'Tickets & WooCommerce', 'event-tickets' ), null, $external_attributes )
+		),
+		( new List_Item( null, null ) )->add_child(
+			new Link( 'https://evnt.is/1ap2', __( 'Creating Tickets', 'event-tickets' ), null, $external_attributes )
+		),
+		( new List_Item( null, null ) )->add_child(
+			new Link( 'https://evnt.is/1ap3', __( 'Event Tickets and Event Tickets Plus Settings Overview', 'event-tickets' ), null, $external_attributes )
+		),
+		( new List_Item( null, null ) )->add_child(
+			new Link( 'https://evnt.is/1ap4', __( 'Event Tickets Plus Manual', 'event-tickets' ), null, $external_attributes )
+		),
+	]
+);
+
+// Build Full Sidebar Content.
+$hero_section->add_section(
+	( ( new Settings_Section() )->add_elements(
+		[
+			// Thank you paragraph.
+			( new Paragraph() )->add_child(
+				new Plain_Text(
+					__( 'Thank you for using Event Tickets! We recommend looking through the settings below so that you can fine tune your specific ticketing needs. Here are some resources that can help.', 'event-tickets' )
```

Comment:
```suggestion
					__( 'Thank you for using Event Tickets! We recommend looking through the settings below so that you can fine-tune your specific ticketing needs. Here are some resources that can help.', 'event-tickets' )
```
"fine tune" is hyphenated when used as a verb.